index.rst

```diff
@@ -13,17 +13,312 @@
+
+.. _PREOPS-1152: https://jira.lsstcorp.org/browse/PREOPS-1152
+.. _RTN-011: https://rtn-011.lsst.io/
+
+Input Data
+==========
+
+The simulation itself is described at https://github.com/lsst-sssc/lsst-simulation .
+It produces the following tables, all of which have reference definitions in the `DPDD`_:
+
+.. _table-ssotables:
+
+.. table:: Major tables to be included in a Solar System DP0.3.  
+
+   +-----------+--------------------------------------------------+--------------+-------------------+---------------------+
+   | Table     | Purpose                                          | Size in rows | Number of columns | Data volume on disk |
+   |           |                                                  |              | (actual/DPDD)     | (in XXX format)     |
+   +===========+==================================================+==============+===================+=====================+
+   | DIASource | Observation in a single difference image         | ~1B          |                   |                     |
+   +-----------+--------------------------------------------------+--------------+-------------------+---------------------+
+   | SSSource  | Apparition of a reconstructed SSO in an image    | ~1B          |                   |                     |
+   +-----------+--------------------------------------------------+--------------+-------------------+---------------------+
+   | SSObject  | Reconstructed SSO with orbital parameters        | ~10M         |                   |                     |
+   +-----------+--------------------------------------------------+--------------+-------------------+---------------------+
+   | MPCORB    | Shadow of Minor Planet Center database           | ~10M         |                   |                     |
+   +-----------+--------------------------------------------------+--------------+-------------------+---------------------+
+   | (total)   |                                                  |              |                   | 0.5 - 1.0 TB        |
+   +-----------+--------------------------------------------------+--------------+-------------------+---------------------+
+
+The simulation is based on a specific, chosen OpSim run, which will be documented.
+The actual simulation output to be used for DP0.3 is yet to be produced, but a prototype dataset exists and has been in use by the SSSC, ingested into a PostgreSQL database.
+
+Possible image-outline table
+----------------------------
+
+As noted above, the existing SSSC data for DP0.3 is based on a catalog simulation,
+with no images involved in the data chain.
+However, the simulation _was_ based on a survey simulation and therefore on pointings and
+sky coverage from that simulation.
+
+It might improve the user experience, as well as allow the exercise of additional RSP capabilities,
+to include a simple visit table, perhaps in ObsCore format, including the pointings and sky coverage of the
+survey on which the catalogs are based.
```

Comment:
I'm a bit wary of dangling a nice-sounding feature if there's nobody who is well-positioned to implement it. Opsim -> ObsCore/Visit table is not a production data path and I'm not aware of any DM code to do that, so it would be "new scope" just for DP0.3 for someone in DM to implement that.

Reply:
@ctslater A few points:

- The only even remotely non-trivial thing about the conversion from OpSim to ObsCore is the computation of the visit outline on the sky from the boresight and rotation.  But this conversion must already have been done in order to execute the SSO simulation, so it seems like code should be available.
- Starting from that point, I'd expect to do the table conversion myself, for this static dataset.
- It is, in fact, an LSST requirement to produce and serve an ObsCore-like table based on scheduler outputs - this is the ObsLocTAP service.  I'll make this explicit in the document.
- The ability to display projected visits as outlines on the sky is not in the DM requirements, but it would be quite odd for us not to provide users with a way to visualize the output of the ObsLocTAP service, and in fact the Portal will do this for free when presented with the resulting table, or a subset of rows from it.

Reply:
I agree it is not a huge amount of work, but it is also not zero work, so we should identify who is responsible for providing it. Just trying to avoid a situation where someone gets volunteered at the last minute.